mimic/model/mailgun_objects.py

```diff
@@ -0,0 +1,119 @@
+"""
+Mailgun object storage
+"""
+import time
+from characteristic import attributes, Attribute
+
+
+@attributes(["message_id", "to", "msg_from", "subject", "body",
+             Attribute("headers", default_factory=dict)])
```

Comment:
Apologies - I was wondering what `headers` here represents.

It looks like it does not get serialized as JSON in `generate_events`, and in fact the headers only consist of to/from/subject/message-id.

But it looks like it does get returned when the someone queries `/messages/header?...`.  I apologize, I couldn't find this in the mailgun documentation - does that return only the non to/from/subject/message-id headers (in all of [these](https://tools.ietf.org/html/rfc4021#section-2))?  Or does it return only the special arbitrary headers that start with X (`h:X-My-Header` values from [the sending docs](https://documentation.mailgun.com/api-sending.html#sending)?